content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -59,7 +59,7 @@ different Kubernetes components.
 | `CSIDriverRegistry` | `false` | Alpha | 1.12 | 1.13 |
 | `CSIDriverRegistry` | `true` | Beta | 1.14 | |
 | `CSIInlineVolume` | `false` | Alpha | 1.15 | - |
-| `CSIMigration` | `false` | Alpha | 1.14 | |
+| `CSIMigration` | `false` | Beta | 1.16 | |
```

Comment:
@davidz627 . 
- The alpha state of the CSIMigration feature gate should stay in the table. Also, would you add the last version for the alpha state in the last column.
- Is the CSIMigration feature gate enabled by default in beta?  If so, the second column in line 62 should be set to `true`. Thanks.

Preview:
https://deploy-preview-15629--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/

Reply:
hi @kbhawkey this feature is actually not going to beta this release. Thanks!